tests/Services/ConvertToJsonServiceTest.php

```diff
@@ -14,13 +14,15 @@ public function testConvert_success()
         $result = \ProjectManager\Services\ConvertToJsonService::convert([1, 2, 3, 4], 0);
         $expected = '{"message":"Successfully retrieved projects","data":[1,2,3,4]}';
         $this->assertSame($expected, $result);
+        $this->assertSame(200, http_response_code());
     }
 
-    public function testConvert_success_unexpected_error()
+    public function testUnexpectedErrorResponse_success()
     {
-        $result = \ProjectManager\Services\ConvertToJsonService::convert([NAN], 0);
+        $result = \ProjectManager\Services\ConvertToJsonService::UnexpectedErrorResponse();
         $expected = '{"message":"Unexpected error","data":[]}';
         $this->assertSame($expected, $result);
+        $this->assertSame(500, http_response_code());
     }
 
     public function testConvert_failure()
```

Comment:
General comment - will need unit test for under 0 handle error

Reply:
logic method has changed. no longer needed